### PR TITLE
dbeaver/dbeaver#21088 Warn overwrite while exporting project

### LINKED ISSUE #21088: Warn of file replacement while exporting project
**Describe the solution you'd like**
When you are exporting a project file, and the name matches an existing file in the selected folder, warn the user that the previous file will be lost.

**Describe alternatives you've considered**

- A pop-up window with the warning message
- Warning message below the output file name

### PR DESCRIPTION
**Pull request by MontejoJorge**
Resolves #21088
First time contributing to this project, for translations to other languages I should use google translate or other contributors who speak those languages will translate them?

### CODE REVIEW

**Comment by LonwoLonwo** [comment]
Hello @MontejoJorge 

Thanks for the contribution.
There is no reason to translate into other languages with the Google Translate. Use only English + your native one (if it is not English).

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/tools/project/ProjectExportWizard.java [hunk 1]
```diff
@@ -119,6 +119,14 @@ private void exportProjects(DBRProgressMonitor monitor, final ProjectExportData
 
         String archiveName = exportData.getArchiveFileName() + ExportConstants.ARCHIVE_FILE_EXT;
         File archiveFile = new File(exportData.getOutputFolder(), archiveName);
+        
+        if (archiveFile.exists()) {
+        	boolean overwrite = DBWorkbench.getPlatformUI().confirmAction(CoreMessages.dialog_project_export_wizard_file_overwrite_window_title, 
+        			NLS.bind(CoreMessages.dialog_project_export_wizard_file_overwrite_confirm, archiveName), true);
+        	
+        	if (!overwrite) return;
+        }
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/tools/project/ProjectExportWizard.java]: Please don't mix tabs and spaces. We use 4 space indents. It's also good to break long expressions into several lines:

```suggestion
        if (archiveFile.exists()) {
            boolean overwrite = DBWorkbench.getPlatformUI().confirmAction(
                CoreMessages.dialog_project_export_wizard_file_overwrite_window_title,
                NLS.bind(CoreMessages.dialog_project_export_wizard_file_overwrite_confirm, archiveName),
                true
            );

            if (!overwrite) {
                return;
            }
        }
```

**Review by E1izabeth — APPROVED** [review]

**Comment by MontejoJorge** [comment]
Sorry about that, i just saw the commit message guidelines

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_es.properties [hunk 2]
```diff
@@ -357,6 +357,10 @@ dialog_project_export_wizard_start_title = Exportar proyecto(s)
 
 dialog_project_export_wizard_window_title = Asistente de exportaci\u00F3n de proyecto
 
+dialog_project_export_wizard_file_overwrite_window_title = El archivo de destino ya existe
+
+dialog_project_export_wizard_file_overwrite_confirm = El archivo"{0}" ya existe. Te gustaria sobrescribirlo?
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_es.properties]:
```suggestion
dialog_project_export_wizard_file_overwrite_confirm = El archivo "{0}" ya existe. Te gustaria sobrescribirlo?
```

**Review by LonwoLonwo — APPROVED** [review]

**Comment by LonwoLonwo** [comment]
Ok, thanks for the changes.
Now we are waiting approve from the QA Team.

**Comment by MontejoJorge** [comment]
Thanks @LonwoLonwo ! if is there any issue that u find kind of easy to solve i would like to try!

**Comment by uslss** [comment]
verified

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**